job_spec/OPERA_DISP_TMS.yml

```diff
@@ -0,0 +1,79 @@
+OPERA_DISP_TMS:
+  required_parameters:
+    - tile_type
+    - bounds
+    - orbit_direction
+    - start_date
+    - end_date
+  parameters:
+      tile_type:
+        api_schema:
+          description: Type of tile map to create
+          type: string
+          minLength: 8
+          maxLength: 15
+          enum:
+            - displacement
+            - secant_velocity
+            - velocity
+          example: displacement
+      bounds:
+        api_schema:
+          description: Integer bounds in format (minlon minlat maxlon maxlat)
+          type: array
+          minItems: 4
+          maxItems: 4
+          example:
+            - -121
+            - 38
+            - -120
+            - 39
+      orbit_direction:
+        api_schema:
+          description: Orbit direction of data
+          type: string
+          minLength: 9
+          maxLength: 10
+          enum:
+            - ascending
+            - descending
+          example: descending
+      start_date:
+        api_schema:
+          description: Start of secondary date range to generate a tile map for (%Y%m%d)
+          type: string
+          minLength: 8
+          maxLength: 8
+          example: 20200101
+      end_date:
+        api_schema:
+          description: End of secondary date range to generate a tile map for (%Y%m%d)
+          type: string
+          minLength: 8
+          maxLength: 8
+          example: 20200814
```

Comment:
There's an extra level of indentation here:
```suggestion
  parameters:
    tile_type:
      api_schema:
        description: Type of tile map to create
        type: string
        minLength: 8
        maxLength: 15
        enum:
          - displacement
          - secant_velocity
          - velocity
        example: displacement
    bounds:
      api_schema:
        description: Integer bounds in format (minlon minlat maxlon maxlat)
        type: array
        minItems: 4
        maxItems: 4
        example:
          - -121
          - 38
          - -120
          - 39
    orbit_direction:
      api_schema:
        description: Orbit direction of data
        type: string
        minLength: 9
        maxLength: 10
        enum:
          - ascending
          - descending
        example: descending
    start_date:
      api_schema:
        description: Start of secondary date range to generate a tile map for (%Y%m%d)
        type: string
        minLength: 8
        maxLength: 8
        example: 20200101
    end_date:
      api_schema:
        description: End of secondary date range to generate a tile map for (%Y%m%d)
        type: string
        minLength: 8
        maxLength: 8
        example: 20200814
```

Reply:
Thanks for the suggestions Jake!